src/goCheck.ts

```diff
@@ -45,6 +45,37 @@ export function notifyIfGeneratedFile(e: vscode.TextDocumentChangeEvent) {
 			}
 		});
 	}
+
+	let settingsBuildTags = goConfig['buildTags'] ? goConfig['buildTags'].split(' ') : null;
+	let fileBuildTags = [];
+	const openSettings = {title: 'Open settings'};
+
+	for (let i = 0; i < e.document.lineCount; i++) {
+		if (e.document.lineAt(i).isEmptyOrWhitespace) {
+			break;
+		}
+		let foundBuildTags = e.document.lineAt(i).text.match(/^\/\/\s*\+build\s+(.+)$/)[1].split(' ');
+		if (foundBuildTags) {
+			foundBuildTags = foundBuildTags.filter(item => item !== '');
+			fileBuildTags.push(...foundBuildTags);
+		}
+	}
+
+	if (fileBuildTags) {
+		for (let i = 0; i < fileBuildTags.length; i++) {
```

Comment:
Take the example of `// +build darwin freebsd netbsd openbsd`
This code here will show the prompt if any of the above is not present in the settings.
But according to https://dave.cheney.net/2013/10/12/how-to-use-conditional-compilation-with-the-go-build-tool,  "a build tag is evaluated as the OR of space-separated options"

So having any one of the above in the build tags is a sufficient.

Also, reading on in the same article, we have , "each option evaluates as the AND of its comma-separated terms" and "A file may have multiple build tags. The overall constraint is the logical AND of the individual constraints"

So looks like a lot more logic needs to be invloved here

Reply:
Okay, I get it, I only have a one question:
If the file has build tags like 
```// +build linux```
```// +build 386```, should it be present in the settings file like
 ```goBuildTags: linux,darwin``` as the goBuildTags holds a string?

Reply:
@urohit011 The `go.buildTags` takes space separated words.

So in your example the setting should be `"go.buildTags": "linux 386"`

If the file has `// +build abc def xyz`, then the build tags setting should contain either `abc`, `def` or `xyz`

Reply:
Hello @ramya-rao-a, thanks for the explanation, In my question what I actually wanted to know was  what should the ```"go.buildTags"``` should look like if the file contains two separate lines such as:
```// +build linux```
```// +build 386```

Reply:
It should look like this: `"go.buildTags": "linux 386"`

Reply:
Got it @ramya-rao-a , working on it